### PR TITLE
refactor: wiki ex uses REST for CRUD

### PR DESCRIPTION
**Pull request by cristianmtr**
- update Jina to `1.1.0`
- modify flow yamls to enable REST API
- add index_rest that indexes docs using the `/index` REST API
- add index_rest to click options
- change README to point towards `/search` (NOT `/api/search`)

### CODE REVIEW

File: wikipedia-sentences/app.py [hunk 1]
```diff
@@ -69,32 +98,38 @@ def dryrun():
...
+                    \n |                                   🤖🤖🤖                                           | \
                     \n | The directory {workspace} already exists. Please remove it before indexing again.  | \
-                    \n |                                   🤖🤖🤖                                         | \
-                    \n +----------------------------------------------------------------------------------+')
+                    \n |                                   🤖🤖🤖                                           | \
+                    \n +------------------------------------------------------------------------------------+'
+            )
+            sys.exit(1)
+
+    print(f'### task = {task}')
```
Comment (rutujasurve94) [inline wikipedia-sentences/app.py:119]: We can remove this `print` statement

File: wikipedia-sentences/app.py [hunk 2]
```diff
@@ -69,32 +98,38 @@ def dryrun():
...
+        index_restful(num_docs)
+    elif task == "query":
         if not os.path.exists(workspace):
             print(f"The directory {workspace} does not exist. Please index first via `python app.py -t index`")
         query(top_k)
-    if task == "query_restful":
+    elif task == "query_restful":
         if not os.path.exists(workspace):
             print(f"The directory {workspace} does not exist. Please index first via `python app.py -t index`")
         query_restful()
```
Comment (rutujasurve94) [inline wikipedia-sentences/app.py:131]: `query_restful` doesn't need `top_k` as the config ymls take care of this.

**Review by rutujasurve94 — APPROVED** [review]
LGTM with minor comments

**Review by bwanglzu — APPROVED** [review]
LGTM!

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Pull request merged**